flaskbb/app.py

```diff
@@ -349,3 +366,28 @@ def after_cursor_execute(conn, cursor, statement,
                                  parameters, context, executemany):
             total = time.time() - conn.info['query_start_time'].pop(-1)
             app.logger.debug("Total Time: %f", total)
+
+
+def load_plugins(app):
+    app.pluggy.add_hookspecs(spec)
+    try:
+        with app.app_context():
+            plugins = PluginRegistry.query.all()
+
+    except OperationalError:
```

Comment:
This is thrown if the table isn't created yet, so upgrades won't bomb. It can potentially be removed later, but I'd need to test a fresh install without it. My gut says the install would fail, but it's been wrong before.